accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java

```diff
@@ -88,15 +89,26 @@ public class AcToolUiService {
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     AcInstallationServiceInternal acInstallationService;
 
-    @Reference(policyOption = ReferencePolicyOption.GREEDY)
-    private WebConsoleConfigTracker webConsoleConfig;
-
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private AcHistoryService acHistoryService;
 
+    @ObjectClassDefinition(name = "AC Tool UI Service", 
+            description="Service that allows to apply AC Tool configuration from the UI and gather some status about the current ACLs of a system")
+    protected static @interface Configuration {
+        
+        @AttributeDefinition(name="Allowed to read", description="Principal names allowed to read status about the current acls maintained in the system")
+        String[] allowReadPrincipalNames() default { "admin" };
+        
+        @AttributeDefinition(name="Allowed to write", description="Principal names allowed to modify the ACLs in the system via ACTool configuration files")
+        String[] allowWritePrincipalNames() default  { "administrators", "admin" };
```

Comment:
TODO: overwrite config for AEMaaCS with `$[env:aemCloudAdministrators;default=administrators]`

Reply:
Done